web/src/components/studies/StudyForm.tsx

```diff
@@ -51,34 +51,34 @@ const convertStudyFormDataToApiRequest = (data: StudyFormData) => {
-    involves_ucl_sponsorship: data.involvesUclSponsorship ? data.involvesUclSponsorship : undefined,
-    involves_cag: data.involvesCag ? data.involvesCag : undefined,
+    involves_ucl_sponsorship: data.involvesUclSponsorship ? data.involvesUclSponsorship : false,
+    involves_cag: data.involvesCag ? data.involvesCag : false,
     cag_reference: data.cagReference ? data.cagReference.toString() : undefined,
-    involves_ethics_approval: data.involvesEthicsApproval ? data.involvesEthicsApproval : undefined,
-    involves_hra_approval: data.involvesHraApproval ? data.involvesHraApproval : undefined,
+    involves_ethics_approval: data.involvesEthicsApproval ? data.involvesEthicsApproval : false,
+    involves_hra_approval: data.involvesHraApproval ? data.involvesHraApproval : false,
     iras_id: data.irasId ? data.irasId : undefined,
-    is_nhs_associated: data.isNhsAssociated ? data.isNhsAssociated : undefined,
-    involves_nhs_england: data.involvesNhsEngland ? data.involvesNhsEngland : undefined,
+    is_nhs_associated: data.isNhsAssociated ? data.isNhsAssociated : false,
+    involves_nhs_england: data.involvesNhsEngland ? data.involvesNhsEngland : false,
     nhs_england_reference: data.nhsEnglandReference ? data.nhsEnglandReference.toString() : undefined,
-    involves_mnca: data.involvesMnca ? data.involvesMnca : undefined,
-    requires_dspt: data.requiresDspt ? data.requiresDspt : undefined,
-    requires_dbs: data.requiresDbs ? data.requiresDbs : undefined,
+    involves_mnca: data.involvesMnca ? data.involvesMnca : false,
+    requires_dspt: data.requiresDspt ? data.requiresDspt : false,
+    requires_dbs: data.requiresDbs ? data.requiresDbs : false,
     is_data_protection_office_registered: data.isDataProtectionOfficeRegistered
       ? data.isDataProtectionOfficeRegistered
       : undefined,
     data_protection_number:
       data.dataProtectionPrefix && data.dataProtectionDate && data.dataProtectionId
         ? `${data.dataProtectionPrefix}/${data.dataProtectionDate}/${data.dataProtectionId}`
         : undefined,
-    involves_third_party: data.involvesThirdParty ? data.involvesThirdParty : undefined,
-    involves_external_users: data.involvesExternalUsers ? data.involvesExternalUsers : undefined,
-    involves_participant_consent: data.involvesParticipantConsent ? data.involvesParticipantConsent : undefined,
+    involves_third_party: data.involvesThirdParty ? data.involvesThirdParty : false,
+    involves_external_users: data.involvesExternalUsers ? data.involvesExternalUsers : false,
+    involves_participant_consent: data.involvesParticipantConsent ? data.involvesParticipantConsent : false,
     involves_indirect_data_collection: data.involvesIndirectDataCollection
       ? data.involvesIndirectDataCollection
-      : undefined,
+      : false,
     involves_data_processing_outside_eea: data.involvesDataProcessingOutsideEea
       ? data.involvesDataProcessingOutsideEea
-      : undefined,
+      : false,
```

Comment:
There are still issues with this approach if the fields are no longer optional:
1. This needs to be reflected in `api.web.yaml`
2. The `go` type for a study may need to be updated as the model currently suggests some of the fields can be `nil` (i.e. optional)
3. The above solution does not address the follow-on questions that require text input, e.g. the `iras_id`
4. Updating the `data_protection_number` field leaves a permanent record in the DB, even if I uncheck `is_data_protection_office_registered ` later on.

Probably a bit more thinking is needed around how this works. Maybe @t-young31 can advise further on representing these values as `go` types?

Reply:
I spotted something else that may be related to this conversation:
- I filled out a study with the `Data Protection Registration Number` filled out,
- I submitted the study and marked it for review,
- I then went to edit the study and changed the `Data Controller` text,
- As I went through the next study pages the `Data Protection Registration Number` was now blank and it wouldn't let me submit until I had filled out the field again.

Weirdly, this only happened when I updated the `Data Controller` input text. Changing other parts of the form didn't produce the above behaviour.

Reply:
the data controller is a required field and if you changed it, that affects the data protection registration number since the latter is based on the former
I think that behaviour is largely expected, unless I'm missing something

Reply:
I see, I wasn't aware of the coupling between the two fields. If they are closely linked I wonder if it would be better to present them on the same form step (for a different PR though)?